### ISSUE TITLE
Parallel Processing is Hard-coded

### ISSUE THREAD

**Issue by DarioS** [larsson-lab/SEISMIC#3]
`mut_effect_cores <- min(detectCores(), 32)` is not good for a server in a statistics deparment with multiple concurrent users.

```
$ lscpu
Architecture:                    x86_64
CPU(s):                          32
```

The YAML file has a parameter named `cores`. Why is it ignored in R code?

**Comment by m-bostrom** [comment]
The `cores` parameter affects the main analysis loop, but as you mention the mutation effect annotation step does not respect this setting. I agree this should not be the case, and will change it to either use the same `cores` value, or add a different parameter for the annotation step. The reason that different approaches to parallelisation were used in the first place was that the main analysis loop used to use a lot more RAM, and using the same number of cores for both steps would then either use too much memory or the annotation step would be slow.

I'll let you know when I've fixed this issue.

**Comment by m-bostrom** [comment]
Fixed in 4653acd68cb1767d1743fabd7ab6e55452e7d64b. All steps that use parallel processing now respect the `cores` parameter.
